orc/src/java/org/apache/orc/DataReader.java

```diff
@@ -53,4 +62,9 @@ DiskRangeList readFileData(
    * @param toRelease The buffer to release.
    */
   void releaseBuffer(ByteBuffer toRelease);
+
+  DataReader clone();
```

Comment:
I'll add a comment, but it does clone the FileHandle so that they can be close separately.